### PR TITLE
[PW-7449] Remove Parent Class from LEM/BalancePlatform

### PR DESCRIPTION
**Pull request by jillingk**
**Description**
Parent Class is not needed here since we have packages. Removed them, added JSON constructors and modified the UT's

### CODE REVIEW

**Comment by jillingk** [comment]
Again don't mind the commits, I merged the BalancePlatform branch herein hence the many commits from that branch.

File: src/main/java/com/adyen/service/legalentitymanagement/Documents.java [hunk 1]
```diff
@@ -4,6 +4,7 @@
 import com.adyen.Service;
 import com.adyen.constants.ApiConstants;
 import com.adyen.model.legalEntityManagement.Document;
+import com.adyen.model.legalEntityManagement.JSON;
```
Comment (michaelpaul) [inline src/main/java/com/adyen/service/legalentitymanagement/Documents.java:7]: Could you also please rename the models package to `com.adyen.model.legalentitymanagement`?

**Review by michaelpaul — COMMENTED** [review]
The main part looks good! I just want to include another change here.

File: src/main/java/com/adyen/service/legalentitymanagement/Documents.java (hunk 1 above)
Comment (jillingk) [reply src/main/java/com/adyen/service/legalentitymanagement/Documents.java]: done!

**Review by AlexandrosMor — APPROVED** [review]

**Review by wboereboom — APPROVED** [review]

**Pull request merged**